src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 
-## CSS Selectors
+    ![](images/wizard-image-17.png)
 
-| **Element** |  **CSS Class** |  **Description**  |
-| ---|---|---
-| .wizard |  .wizard.wizard-vertical|  When the wizard orientation is vertical  |
-| .wizard-item |  .wizard-item.active|  Defines the current active step  |
-| .wizard-item |  .wizard-item.past|  Defines the previous step  |
-| .wizard-item |  .wizard-item.next|  Defines the next step  |
+**Add Content to your Wizard**
 
-## Advanced Use Case
+1. Create a condition to control what is displayed, depending on what step is active.
+    
+    ![](images/wizard-image-19.png)
 
-### Use Wizard with ListRecords
+1. Add the relevant content to each for each of the steps.
 
-1. Drag the Wizard Pattern into the screen.
+     The following example shows the Shipping Details step that contains text and input boxes.
 
-1. In the Content placeholder, drag a ListRecords widget.
+      ![](images/wizard-image-18.png)
 
-1. Set the Line Separator parameter of the ListRecords to None.
+1. Create variables for each of the elements in the step container. 
+    
+    ![](images/wizard-image-20.png)
 
-1. In the ListRecords widget, drag a WizardItem Pattern.
+**Create a Screen Action**
 
-1. In the WizardItem Pattern, use expressions to display the database content you need.
+To ensure that all of the information the user enters is passed from step to step, you must create a screen action for the **Previous** and **Next** navigation buttons.
 
-1. Publish and test.
+1. Select the **Previous** button.
+1. On the **Properties** tab, from the **Destination** drop-down, select **(New Screen Action)**.
 
-### Create a Wizard navigation
+    ![](images/wizard-image-22.png)
 
-Use this example to create a three steps Wizard with continue and back buttons.
+1. From the Toolbox, drag the Assign block onto your screen and set the properties. This ensures that the user inputs get passed from step to step.
 
-1. Drag the Wizard Pattern into the page. By default, it comes with three WizardItems.
+    ![](images/wizard-image-21.png)
 
-1. Create a Local Integer Variable, named WizardStepIndex, to hold the current Wizard step.
+1. Repeat these steps for the **Previous** button.
 
-1. Drag two Button widgets and set their names to Continue and Back.
+After following all of the steps in each of the sections, you can publish the module, and test the Wizard in your app.
 
-1. Create a new Screen Action named WizardNavigation.
+## Properties
 
-    ![](images/wizard-image-4.png)
+### Wizard
 
-1. Create a mandatory boolean Input Parameter on this action named IsNext.
+| **Property** |  **Description** |  **Usage** | 
+|---|---|---|
+| Orientation (Orientation Identifier): Optional  |  Set the wizard orientation. |<ul><li>_Entities.Orientation.Horizontal_ - The wizard displays horizontally </li></ul> <ul><li>_Entities.Orientation.Vertical_ - The wizard displays vertically</ul></li>
```

Comment:
The dropdown list will show the different orientation options: (...)
--- also, we need to state what's the default if it's blank (I imagine it's Horizontal, but we need to confirm that by trying it out)

Reply:
Edited:
Property | Description | Usage
-- | -- | --
Orientation (Orientation Identifier): Optional | Set the wizard orientation. By default the Wizard displays horizontally | The Orientation drop-down shows the different orientation options: *Horizontal (Entities.Orientation.Horizontal) - The wizard displays horizontally * Vertical (Entities.Orientation.Vertical) - The wizard displays vertical